service/profile_master_service.go

```diff
@@ -0,0 +1,53 @@
+package service
+
+import (
+	"context"
+
+	"github.com/Kotlang/authGo/db"
+	pb "github.com/Kotlang/authGo/generated"
+	"github.com/SaiNageswarS/go-api-boot/auth"
+	"github.com/SaiNageswarS/go-api-boot/logger"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type ProfileMasterService struct {
+	pb.UnimplementedProfileMasterServer
+	db *db.AuthDb
+}
+
+func NewProfileMasterService(authDB *db.AuthDb) *ProfileMasterService {
+	return &ProfileMasterService{
+		db: authDB,
+	}
+}
+
+func (s *ProfileMasterService) GetProfileMaster(ctx context.Context, req *pb.GetProfileMasterRequest) (*pb.ProfileMasterResponse, error) {
+	_, tenant := auth.GetUserIdAndTenant(ctx)
+
+	language := req.Language
+	if language == "" {
+		return nil, status.Error(codes.InvalidArgument, "Language not specified")
+	}
+
+	profileMasterListChan, profileMasterListErrorChan := s.db.ProfileMaster(tenant).FindByLanguage(req.Language)
+	list := make([]*pb.ProfileMasterProto, 0)
+
+	select {
+	case profileMasterList := <-profileMasterListChan:
+		for _, profileMaster := range profileMasterList {
```

Comment:
Use "github.com/jinzhu/copier" to copy from model to proto directly. No need to manually copy model to proto.